### DISCUSSION TITLE
Articles by authors

### DISCUSSION THREAD

**Discussion by leoalv** [joomla/joomla-cms#31700] [2. Cold Features]
We have known for a long time that Joomla articles are very limited, that was partly solved with the arrival of tags and then the extra fields. But a chronic and most requested need is to be able to show galleries articles by an author.

An author page is necessary with the list of his articles, as if other CMS have it.

**Is there a plan to integrate this?**

**Comment by drmenzelit** [comment]
It is already possible to create an author's page using the contact component.
Some weeks ago Daniel Dubois and me created some tutorials for this: 
https://www.web-eau.net/blog/how-to-create-an-author-page-in-joomla
https://www.dr-menzel-it.de/blog/override-challenge-autor-seite

**Comment by leoalv** [comment]
@drmenzelit  Thanks for your answer, I know both solutions, but they are very limited; You could even override the com_content category, but it has its limitations as well.

The idea is to create an author page with the possibility of paging their articles, without too many complications or unnecessary queries to the database.

This is good for newspaper sites, magazines, and even blogs. The ones I've known have always moved from com_content to K2. Mainly they go to K2 for the author page and the possibility of cutting images according to your needs.

**My opinion is to create a new view and AUTHOR model in com_content.**

Please I need that to be requested as a new feature and its possibilities discussed.

**Comment by mwdev25** [comment]
Joomla desperately needs this function. Essentially (1) a category blog filtered/assigned by author and then (2) an option to redirect author link in articles/categories to this page. 

Such function is a must-have on any content-centric website with multiple authors (news, magazine, education, etc). 

There are a couple 3rd party templates that have custom overrides but this is such a seemingly basic function - it'd be great if we could have this in the core. 

I agree with @leoalv that other CMS (we shall not name) have this and is one thing contributing to selection over Joomla in some cases. Namely, every single client wanting a blog /news website.
